frontend/src/Home.jsx

```diff
@@ -1,43 +1,49 @@
 import React, { Component } from "react";
+import PropTypes from "prop-types";
 import LOCALIZE from "./text_resources";
 import ContentContainer from "./components/commons/ContentContainer";
 import LoginTabs from "./components/authentication/AuthenticationTabs";
+import Dashboard from "./Dashboard";
 import { Helmet } from "react-helmet";
+import { connect } from "react-redux";
 
 class Home extends Component {
-  //TODO(fnormand): Remove this part when implementing login functionality in the backend
-  //===========================================
-  state = {
-    isAuthenticated: false
+  static propTypes = {
+    // Props from Redux
+    authenticated: PropTypes.bool
   };
 
-  authentification = () => {
-    this.setState({ isAuthenticated: true });
-  };
-  //===========================================
-
   render() {
     return (
       <div className="app">
         <Helmet>
           <title>{LOCALIZE.titles.home}</title>
         </Helmet>
         <ContentContainer>
-          <h1>{LOCALIZE.homePage.title}</h1>
-          <p>{LOCALIZE.homePage.description}</p>
-          <div>
-            <LoginTabs authentification={this.authentification} />
-          </div>
-          {this.state.isAuthenticated && (
+          {!this.props.authenticated && (
             <div>
-              <h3>Welcome!</h3>
-              <p>You've just logged in.</p>
+              {" "}
```

Comment:
This empty space will get announced to a screen reader.

Reply:
Already fixed in previous commits.